test/runtime/connectivity.go

```diff
@@ -336,10 +336,6 @@ var _ = Describe("RuntimeValidatedConntrackTest", func() {
 			"could not get metadata for container %q", curl2ContainerName)
 		By(fmt.Sprintf("httpd1 container Docker networking: %s", curl2DockerNetworking))
 
-		By("Showing policies imported to Cilium")
-		res := vm.PolicyGetAll()
-		fmt.Fprintln(ginkgo.GinkgoWriter, res.CombineOutput())
```

Comment:
Why is this removed? This is useful debugging information. Having such information in the logs makes it easier to triage and debug issues as a developer.

Reply:
The information will be in the logs, but not per each time that test is executed.. It's too verbose.

Reply:
What makes it verbose? Logging state while the test is running is useful for me as a developer. Before a test is ran, I want a view of what the state of policy is within Cilium. This is what we did in the old Bash tests, and I think it proved highly useful.

Reply:
Fixed